maro/cli/inspector/env_data_process.py

```diff
@@ -0,0 +1,201 @@
+        if not os.path.exists(os.path.join(source_path, Gfiles.name_convert)):
+            raise CliException("Have to regenerate data. Name Conversion File is missed. ")
+
+        if scenario == GlobalScenarios.CIM:
+            if not os.path.exists(os.path.join(source_path, Gfiles.ports_sum)):
+                raise CliException("Have to regenerate data. Summary File is missed. ")
+        elif scenario == GlobalScenarios.CITI_BIKE:
+            if not os.path.exists(os.path.join(source_path, Gfiles.stations_sum)):
+                raise CliException("Have to regenerate data. Summary File is missed. ")
+
+    launch_dashboard(source_path, scenario, epoch_num, prefix)
+
+
+def _init_csv(file_path: str, header: list):
+    """Clean & init summary csv file.
+
+    Args:
+        file_path (str): summary file path.
+        header (list): expected header of summary file.
+    """
+    if os.path.exists(file_path):
+        os.remove(file_path)
+    with open(file_path, "w") as csvfile:
+        writer = csv.DictWriter(csvfile, fieldnames=header)
+        writer.writeheader()
+
+
+def _summary_append(
+        scenario: enumerate, dir_epoch: str,
+        file_name: str, header: list,
+        sum_dataframe: pd.DataFrame, i: int, output_path: str):
+    """Calculate summary info and generate corresponding csv file.
+    To accelerate, change each column into numpy.array.
+
+    Args:
+        scenario (str): Current scenario.
+        This parameter is useless right now. Cause only scenario-cim needs operations within this function.
+        dir_epoch (str): Current epoch.
+        Loop is outside this function. This function calculate a summary within an epoch each time.
+        file_name (str): Name of file needed to be summarized.
+        Some scenario has multiple files within one epoch. e.g.cim.
+        header (list): List of columns needed to be summarized.
+        sum_dataframe (dataframe): Temporary dataframe to restore results.
+        i (int): Index (to indicate the progress in loop).
+        output_path (str): Path of output CSV file.
+
+    """
+    input_path = os.path.join(dir_epoch, file_name)
+    data = pd.read_csv(input_path)
+    data_insert = []
+    for ele in header:
+        data_insert.append(np.sum(np.array(data[ele]), axis=0))
+    sum_dataframe.loc[i] = data_insert
+    sum_dataframe.to_csv(output_path, header=True, index=True)
+
+
+def _generate_summary(scenario: enumerate, source_path: str, prefix: str):
+    """Generate summary info of current scenario.
+    Different scenario has different data features.
+    e.g. cim has multiple epochs while citi_bike only has one.
+    Each scenario should be treated respectively.
+
+    Args:
+        scenario (enumerate): Current scenario.
+        source_path (str): Data folder path.
+        prefix (str): Prefix of data folders.
+
+    """
+    ports_header = ["capacity", "empty", "full", "on_shipper", "on_consignee", "shortage", "booking", "fulfillment"]
```

Comment:
Since you already use ``maro/cli/inspector/params.py``, why not define the headers in this file?